### ISSUE TITLE
Ability to scroll / pan workspace using keyboard

### ISSUE THREAD

**Issue by microbit-robert** [google/blockly#10182]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Problem

Keyboard users cannot (as far as I'm aware) scroll / pan the workspace without using a mouse. There is an automatic scroll into view feature when the focused block changes (e.g., focus a block outside of the current viewable area). However, this doesn't cover the use case where the user simply wants to look around without changing the currently the focused block.

### Request

It would be useful to allow keyboard users to scroll / pan the workspace so they can look around without changing what block is currently focussed. Perhaps this can be implemented with a modifier key with arrows (shift + arrows?). 

### Alternatives considered

There was a previous suggestion that when the workspace itself is focused, the arrow keys could scroll / pan, but I think this isn't feasible now that the workspace is a position and using the up / down arrows moves you to the next / previous block.
